frontend/src/bookings/calendar.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div>
+    <div class="title inline-title">{{ title }}</div>
+    <keep-alive>
+      <full-calendar ref="qcalendar"
+                     key="qcalendar"
+                     class="calendar-margins"
+                     @view-render="viewRender"
+                     @event-render="eventRender"
+                     :events="events"
+                     :config="configuration"></full-calendar>
+    </keep-alive>
+  </div>
+
+</template>
+
+<script>
+  import { mapState, mapMutations, mapActions } from 'vuex'
```

Comment:
Is there a standard order for importing these? I'm guessing alphabetized should be standard?